tests/testflows/aes_encryption/docker-compose/clickhouse-service.yml

```diff
@@ -0,0 +1,28 @@
+version: '2.3'
+
+services:
+  clickhouse:
+    image: yandex/clickhouse-integration-test
```

Comment:
It is better to move all docker compose files to testflows image. It makes possible to update images inside same PR and not use latest version.
It is better to pass tag directly from runner to `yandex/clickhouse-integration-test` via env. It is possible to add dependency testflows<-integration to CI and your task will receive tag from PR build.

Reply:
The same here. Let's move this to be part of restructuring `test/testflows` folder in a separate PR.